reference/features/distinct.md

```diff
@@ -1,16 +1,16 @@
 # Distinct attribute
 
-The **value** of a field whose attribute is set as a distinct attribute will always be **unique** in the returned documents.
+A distinct attribute is a field whose value is shared by various documents. It is most commonly used to prevent MeiliSearch from returning several instances of similar documents.
 
-In such a case, there will **never be two, or more, occurrences of the same value** of that field in the different documents returned by MeiliSearch.
+The value of a field configured as a distinct attribute will always be unique among returned documents. This means there will **never be more than one occurrence of the same value** of a distinct attribute field in the different documents returned by MeiliSearch.
```

Comment:
```suggestion
The value of a field configured as a distinct attribute will always be unique among returned documents. This means that among all the different documents returned by MeiliSearch, **there will never be more than one occurrence of the same value attached to the distinct attribute**.
```

Reply:
This is a tricky concept to describe (but very easy to demonstrate). Not sure if what I've written is clear; feel free to take or leave it.

Reply:
Yeah, it's difficult to put it in words. I'm not sure either of our attempts make it much clearer, to be honest. 

```suggestion
The value of a field configured as a distinct attribute will always be unique among returned documents. This means **there will never be more than one occurrence of the same value** in the distinct attribute field among the returned documents.
```